packages/emotion-primitives/src/splitProps.js

```diff
@@ -0,0 +1,57 @@
+import {
+  textProps,
+  imageProps,
+  viewProps,
+  textStyleProps,
+  viewStyleProps,
+  imageStyleProps
+} from './props'
+
+function isValidStyleProp(element, propName) {
+  if (element === 'Text') {
+    return textStyleProps.includes(propName)
```

Comment:
`includes` requires polyfills to be used by library's users if they want to support older browsers, not sure if it's worth using in that case as `indexOf` equivalent is just a little bit more verbose

Reply:
Agree! I will replace this with indexOf > -1